tests/distributed/test_pipeline_parallel.py

```diff
@@ -34,7 +37,9 @@ def test_compare_tp(TP_SIZE, PP_SIZE, EAGER_MODE, CHUNKED_PREFILL, MODEL_NAME):
         "--dtype",
         "bfloat16",
         "--tensor-parallel-size",
-        str(max(TP_SIZE, 2)),  # use at least TP_SIZE=2 to hold the model
+        # Use the same number or at most 8 GPUs to hold the model.
+        # In this test we assume the model can fit in 8 GPUs.
+        str(min(TP_SIZE * PP_SIZE, 8)),
```

Comment:
it's not going to work. this will run in multi-node tests with mp backend, and we can use at most 2 GPUs.

you can revert this change, keep it only for your local testing.

Reply:
Reverted with comments.